index.html

```diff
@@ -176,5 +176,33 @@ <h3>Skills</h3>
         </li>
       </ul>
     </section>
+
+    <section id="form-section">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like
+        to chat please get in touch.
+      </h2>
+
+      <form class="" action="index.html" method="post">
+        <div class="fullname">
+          <input class="font-style" type="text" name="user-name" placeholder="Full name" maxlength="30" required />
+          <input class="font-style" type="text" name="user-lastname" placeholder="Last Name" maxlength="30" required id="lastname"/>
+        </div>
+        <input class="font-style" type="email" name="user-email" placeholder="Email address" required />
+        <textarea class="font-style" name="user-message" placeholder="Hey I'd like to connect to..." maxlength="500"></textarea>
+        <button type="submit" name="message">Get in touch</button>
+      </form>
+    </section>
+
+    <footer>
+      <a href="/" class="footer-mail">floperh@gmail.com</a>
+      <ul class="list-format font-style footer-icon">
+        <li><a href="https://github.com/flov3rh" class="fa-brands fa-square-github"></a></li>
+        <li><a href="https://www.linkedin.com/in/flov3rh/" class="fa-brands fa-linkedin"></a></li>
+        <li><a href="#" class="fa-brands fa-angellist"></a></li>
+        <li><a href="https://twitter.com/flov3rh" class="fa-brands fa-twitter"></a></li>
+        <li><a href="mailto: floperh@gmail.com" class="fa-solid fa-envelope"></a></li>
+      </ul>
+    </footer>
```

Comment:
- [ ] I would suggest you please stick to the design as much as possible like font, colours, images, and spacing (margin/padding). As I notice that you design is missing the small background image please implement it and also I would suggest you please download the social media icons from Figma design and use it.
- [ ] Also make sure that your font matches the design as it has more font weight for your email.
### Your design
![his](https://github.com/flov3rh/portfolio/assets/81727988/d17d7790-5aa9-4333-a202-df6a6a25d5d2)

### Figma design
![fimga](https://github.com/flov3rh/portfolio/assets/81727988/a77c1a94-6eec-473d-8576-9ba7d0fbca32)